### DISCUSSION TITLE
Checkmarks for finished games ?

### DISCUSSION THREAD

**Discussion by XeLo8** [OnionUI/Onion#1125] [Ideas]
Hello, yesterday i my mm+ finally arrived and i started playing something opened atari2600 and played adventure... enjoyed it and finished it... wanted to move on other game but i would really like some kind of checkmark or at least something like 
A̶d̶v̶e̶n̶t̶u̶r̶e̶                       Adventure ✔
Example                          Example     X
Example                          Example     X
Example                          Example     X                     
OR which sounded like most natural way to implement this is currently in front of every game i have - so 

-adventure                                                                             +adventure
-platform                                                                               -platform
-puzzle                                                                                   -puzzle                    
-example              if i played them and it would show          +example

**Comment by Rahhide** [comment]
check this out links https://github.com/schmurtzm/Miyoo-Mini-Collection-Switcher
its provides list switcher
